api/History.json

```diff
@@ -0,0 +1,140 @@
+{
+  "api": {
+    "History": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/History",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": true
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": {
+            "version_added": true
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "modify_state": {
+        "__compat": {
+          "description": "<code>pushState</code> and <code>replaceState</code>",
+          "support": {
+            "webview_android": {
+              "version_added": "2.2"
```

Comment:
Change to `true`. Webview follows Chrome numbering not Android numbering. I don't have accurate information that far back.

Reply:
Thanks for the review, I have just pushed a fix.